cicd/Jenkins/Jenkinsfile

```diff
@@ -1246,3 +1276,261 @@ def runAdoptionStats() {
     jslAdoptionMain('reports/ci_report/xunitfinal.xml')
   }
 }
+
+def createBranchChiVue() {
+  withCredentials([usernamePassword(credentialsId: env.GITHUB_TOKEN_CREDENTIALS, usernameVariable: 'USERNAME', passwordVariable: 'ACCESS_TOKEN')]){
+    def projectJson = readJSON file: 'src/chi-vue/package.json', returnPojo: true;
+    def version = projectJson["version"];
+    def versionReturn = projectJson["version"];
+    sh(script: """
```

Comment:
Please, use jslGitHubCloneRepo function.

Reply:
I've added it to both the chi-vue and chi creations